### PR TITLE
Update to bootstrap 4.6.0

### PR DESCRIPTION
**Pull request by barton2526**
This is the latest LTS release

### CODE REVIEW

**Comment by barton2526** [comment]
@RoboticMind

File: assets/css/bootstrap-4.6.0.min.css [hunk 1]
```diff
@@ -0,0 +1,7 @@
...
+/*# sourceMappingURL=bootstrap.min.css.map */
```
Comment (RoboticMind) [inline assets/css/bootstrap-4.6.0.min.css:7]: Minor, but I noticed my browser 404ing when trying to find the map file
```suggestion
/*# sourceMappingURL=bootstrap-4.6.0.min.css.map */
```

**Review by RoboticMind — COMMENTED** [review]
Looks good overall. Didn't spot any major issues - just one minor one that's a one line fix. Tested it locally and didn't see any issues. Read over the changelogs for the version between and didn't spot anything that looks like it'd be an issue

File: assets/js/core/bootstrap-4.6.0.bundle.min.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+//# sourceMappingURL=bootstrap.bundle.min.js.map
```
Comment (RoboticMind) [inline assets/js/core/bootstrap-4.6.0.bundle.min.js:7]: Also meant to mention this one
```suggestion
//# sourceMappingURL=bootstrap-4.6.0.bundle.min.js.map
```

**Review by RoboticMind — APPROVED** [review]
Looks good now

**Pull request merged**